framework/src/controller/channels/child_process_channel.js

```diff
@@ -175,7 +175,7 @@ class ChildProcessChannel extends BaseChannel {
 		});
 	}
 
-	async cleanup() {
+	cleanup() {
```

Comment:
This was kept `async` as pattern to avoid future refactoring. In future we may need to update cleanup logic with async task, then we don't need to update function usage everywhere. Also this pattern should be followed if we or anyone else create different channel.